packages/jest-runtime/src/index.ts

```diff
@@ -1086,6 +1086,19 @@ class Runtime {
         formatStackTrace(stack, this._config, {noStackTrace: false}),
     );
   }
+
+  private constructInjectedModuleArguments() {
```

Comment:
Can we have a test that assures correct params are passed from jest-runtime to @jest/transform? Or is it covered by some e2e test already?

Reply:
It's covered by basically all the tests we have, otherwise `require`, `module`, `jest` etc would not be defined when running tests. I'll try to add a unit test, though 🙂

Reply:
Only if breaking this would result in some kind of very confusing errors, that are super hard to figure out :)

Reply:
hmm, I'm unable to write a good test... A bunch of other tests would fail though, especially the `extraGlobals` tests

Reply:
@thymikee thoughts on the commit I just pushed?